src/Chronos.hs

```diff
@@ -2660,3 +2659,37 @@ aesonParserOffset :: Text -> AET.Parser Offset
 aesonParserOffset t = case decodeOffset OffsetFormatColonOn t of
   Nothing -> fail "could not parse Offset"
   Just x -> pure x
+
+-- | Holds all of the parts encoded by a 'Time'.
+--   Can be used for formatting if what is presently in the API
+--   does not suffice.
+data TimeParts = TimeParts
+  { timePartsDay :: !Int
+  , timePartsMonth :: !Int
```

Comment:
Just one thing: can we add a documentation whether `Month` is started from 0 or from 1?